### DISCUSSION TITLE
Allow custom variables for the permission filters

### DISCUSSION THREAD

**Discussion by Attacler** [directus/directus#5394] [Feature Requests]
It would be nice to have a way (in a extension or something) to add other variables  to the permission rules.
For example the domain that the request is comming from, that would help with stuff like multiple sites based on 1 directus instance.

**Comment by h2bear** [comment]
I would also love the ability to add custom dynamic variables based on my own collections 👍

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
